setup.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python
+
+from distutils.core import setup
+
+setup(name='majorkirby',
+      version='1.0.0',
```

Comment:
Maybe not 1.0 yet? Don't want to send the wrong signal to people who come across this necessarily